src/contrib/cluster/Akka.Cluster.Sharding/reference.conf

```diff
@@ -184,8 +184,10 @@ akka.cluster.sharding {
   distributed-data {
     # minCap parameter to MajorityWrite and MajorityRead consistency level.
     majority-min-cap = 5
-
-    durable.keys = ["shard-*"]
+    
+    # Uncomment this line if you want to use the durable version of DData and make sure that
+    # you also reference the Akka.DistributedData.LightningDB nuget package.
+    # durable.keys = ["shard-*"]
```

Comment:
Make sure we tell users to add this setting back in the documentation (in the other PR you're working on)